src/modules/i18n/locales/en.json

```diff
@@ -123,8 +123,8 @@
     "draw_next_measure": "Click to add another point (double-click to finish)",
     "draw_panel_title": "Draw & Measure on map",
     "draw_point": "Point",
-    "draw_point_description": "Create point",
-    "draw_point_instructions": "Create point",
+    "draw_point_description": "",
+    "draw_point_instructions": "import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'",
```

Comment:
While working on `mf-geoadmin3` I saw this weird change coming there too, I updated/reverted this change in the spreadsheet, you might want to update the translation on your PR too :wink:

Reply:
Done